components/newBillHomePageButtons.jsx

```diff
@@ -0,0 +1,10 @@
+export default function newBillHomePageButtons({text, iconName}){
```

Comment:
There should be a space between the deconstructed fields and after the parameter definitions like `({ text, iconName }) {`